WordPress/src/main/res/values/strings.xml

```diff
@@ -2303,6 +2303,12 @@
     <string name="reader_save_posts_locally_dialog_title">Save Posts for Later</string>
     <string name="reader_save_posts_locally_dialog_message">Save this post, and come back to read it whenever you\'d like. It will only be available on this device — saved posts don\'t sync to your other devices.</string>
 
+    <!-- Reader tags feed -->
+    <string name="reader_tags_feed_see_more_from_tag">More from %s</string>
+    <string name="reader_tags_feed_error_title">No posts found for %s</string>
+    <string name="reader_tags_feed_error_description">This tag might not have any posts, or there was no internet connection.</string>
```

Comment:
This error message is too generic, as it combines "no posts found in tag" (which is a valid condition) with a generic error. Can we make it clearer whether it's a real error or simply no posts found?

Reply:
Makes sense. There is at least one more scenario: when there was an error with the request. Should we have three different messages (one for when posts list returns empty, one for no internet connection and another for a failed request)? cc @osullivanchris 

Also, if the post list returns empty from the back-end, does it make sense to provide a retry button? In theory it's going to return an empty post list again.

Reply:
I think two errors is enough as it might be difficult and not meaningful to distinguish between when the user had no internet connection, and when we failed to fulfil the request. 

Something like:

- Posts failed to load: we couldn't load posts from this tag right now
- No posts for this tag: we couldn't find any posts tagged TagName right now

Reply:
cc @wargcm @dvdchr

Reply:
> Posts failed to load: we couldn't load posts from this tag right now

If we already have some posts locally under a certain tag, what do you think about displaying them instead of showing an error?